src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+  const SEX_FEMALE = 'f';
+
+  return (
+    <>
+      <section className="Person">
```

Comment:
```suggestion
    <section className="Person">
```
React fragment is redundant here as you already have a container